### PR TITLE
added test for cassandra-10711; empty batch causing exception

### PR DESCRIPTION
**Pull request by nutbunnies**

### CODE REVIEW

File: batch_test.py [hunk 1]
```diff
@@ -7,11 +7,24 @@
...
+        """
+        @jira_ticket CASSANDRA-10711
+        """
+        session = self.prepare()
+        session.execute("""
+            BEGIN BATCH
+            APPLY BATCH;
+        """)
+        for node in self.cluster.nodelist():
+            self.assertEquals(0, len(node.grep_log_for_errors()))
```
Comment (ptnapoleon) [inline batch_test.py]: This happens automatically in node teardown, and thus is redundant, no?

Comment (nutbunnies) [inline batch_test.py]: True, but I have an aversion to a test without any assertions.  Could be replaced with a comment I suppose.

Comment (mambocab) [inline batch_test.py]: I prefer the assertion to a comment, personally.

**Comment by mambocab** [comment]
I'm +1.

**Pull request merged**